test/helpers/enums.js

```diff
@@ -2,10 +2,20 @@ function Enum(...options) {
   return Object.fromEntries(options.map((key, i) => [key, web3.utils.toBN(i)]));
 }
 
-module.exports = {
-  Enum,
-  ProposalState: Enum('Pending', 'Active', 'Canceled', 'Defeated', 'Succeeded', 'Queued', 'Expired', 'Executed'),
-  VoteType: Enum('Against', 'For', 'Abstain'),
-  Rounding: Enum('Floor', 'Ceil', 'Trunc', 'Expand'),
-  OperationState: Enum('Unset', 'Waiting', 'Ready', 'Done'),
-};
+function EnumBigInt(...options) {
+  return Object.fromEntries(options.map((key, i) => [key, i]));
+}
```

Comment:
This is actually not BigInt and just regular number. Still, I think this is the way we should implement it and we can have a single version.

I'll update.